core/pkg/resourcehandler/k8sresourcesutils.go

```diff
@@ -24,6 +26,7 @@ var (
 	KubeProxyInfo                = "KubeProxyInfo"
 	ControlPlaneInfo             = "ControlPlaneInfo"
 	CloudProviderInfo            = "CloudProviderInfo"
+	CNIInfo                      = "CNIInfo"
```

Comment:
we are not missing here 
ListRolePolicies = "ListRolePolicies" ?

Reply:
No. ListRolePolicies was used as an example in the tests as a resource we aren't using